pages/Category.vue

```diff
@@ -455,17 +454,11 @@ export default {
     }
   },
   mounted () {
-    this.unsubscribeFromStoreAction = this.$store.subscribeAction(action => {
```

Comment:
Are you sure about this part? Have you test it manually?

Reply:
Yes, I have tested it. Actually, on the initial load of the category page, only two specific actions are getting subscribed so, I think this part is of no use.